### ISSUE TITLE
[USERSNAP] Pioneer keeps telling me proposals are dangerous

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#2701]
Validate

- [x] Add "Do not show this message again" checkbox
- [x] Show proposal modal only once, store "viewed" state **in local storage.**

____________

**Taken at**: https://dao.joystream.org/#/proposals/current
**Fields**:
- Houston, we have a problem.: Pioneer keeps telling me proposals are dangerous
- Give us some details: I am aware.
- No idea. Where is it saved?
- Discord Handle: l1dev

![SCREENSHOT](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/882b4da7-e887-4bf8-8854-e7030af406d8-annotated_987d828d-3c0b-4d54-9c25-a23db3755898.png?etag=f1dd8e59223385634ea08e78ba759c22)
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/882b4da7-e887-4bf8-8854-e7030af406d8-annotated_987d828d-3c0b-4d54-9c25-a23db3755898.png?etag=f1dd8e59223385634ea08e78ba759c22)*

**Browser**: Chrome 99 (Linux)
**Screen size**: 1920x1080
**Browser size**: 2390x1216
**Labels**: bug-non-critical
**[Open #154 in Usersnap](https://app.usersnap.com/l/feedback/ed7e2583-d20d-48e9-9c6c-5030138784f3?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

**Comment by singulart** [comment]
@ivanturlakov I suggest opening a separate issue as per your observation.

**Comment by dmtrjsg** [comment]
@stepien4soft  Here I'd suggest to swap checkboxes, and place "do not show this again" below, to follow the convention :)

**Comment by singulart** [comment]
@dmtrjsg this is sent to DEV despite `qa-passed`? Will make my life harder when compiling the report

**Comment by dmtrjsg** [comment]
@singulart this is a 1min thing, no need to get re-tested we will take care of bringing this further to prod.

**Comment by dmtrjsg** [comment]
@singulart also since this was not specified by myself, the QA verdic is correct, and no impact on qa-reports!
